vscode/test/integration/commands.test.ts

```diff
@@ -0,0 +1,90 @@
+import * as assert from 'assert'
+
+import * as vscode from 'vscode'
+
+import { afterIntegrationTest, beforeIntegrationTest, getFixupTasks, getTranscript, waitUntil } from './helpers'
+
+suite('Commands', function () {
+    this.beforeEach(() => beforeIntegrationTest())
+    this.afterEach(() => afterIntegrationTest())
```

Comment:
No need for the extra closure here.

```suggestion
    this.beforeEach(beforeIntegrationTest)
    this.afterEach(afterIntegrationTest)
```